web/src/components/AppBar/ApiKeyItem.vue

```diff
@@ -1,9 +1,10 @@
 <template>
   <v-list-item>
     <v-list-item-action class="mr-2">
+      <!-- stop closing the menu if API key refreshed -->
```

Comment:
This is good, but since HTML doesn't allow us to put comments inside elements, it may not be clear whether "stop" is referring the effect or the actual `.stop` modifier (and it's both).

What do you think of:
```suggestion
      <!-- ".stop" prevents closing the parent v-menu when this is clicked -->
```